### PR TITLE
Added new blog post on version 1.2.x improved plugin management.

### PR DESCRIPTION
**Pull request by ldimaggi**

### CODE REVIEW

**Comment by EricWittmann** [comment]
@kahboom Could you have a quick look at this in jekyll just to make sure it looks OK and there aren't any broken links or anything?

File: _blog-src/_posts/2016-02-22-plugin-mgmt.markdown [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+To this: request.getHeaders().put("Test-Policy", "quite true");
+
+And then, rebuild the plugin with this command:
+
+```shell
+mvn clean install -DskipTests
+```
+
+(The “skipTests” directive is a bit of laziness. There is a test included in the plugin that will fail because of the change that we just made. You can either run this command as it, or you can edit the test in the plugin to also look for a string of “quite true.”)
```
Comment (kahboom) [inline _blog-src/_posts/2016-02-22-plugin-mgmt.markdown]: I think you meant, `You can either run this command as is` here.

**Comment by kahboom** [comment]
@ldimaggi - This is a great, detailed post that's easy to follow. Thanks for always showing the blog a little bit of love. :D It looks great in the UI, the only thing is that there was a very minor typo if you'd like for me to nitpick, which I commented on. Other than that, looks good. :+1: 


cc @EricWittmann

File: _blog-src/_posts/2016-02-22-plugin-mgmt.markdown (hunk 1 above)
Comment (EricWittmann) [inline _blog-src/_posts/2016-02-22-plugin-mgmt.markdown]: Good catch.  :)  I'll merge and then fix it so that len doesn't have to update the PR.

**Pull request merged**